src/pages/CheckReservation/CheckReservation.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from 'react';
+import ReservationTicket from './ReservationTicket';
+import S from './Styled.CheckReservation';
+
+const CheckReservation = () => {
+  const [reservationData, setReservationData] = useState([]);
+  const [currentTab, setCurrentTab] = useState('예정된 여행');
+
+  useEffect(() => {
+    fetch('/data/Reservation_Data-1.json')
+      .then(res => res.json())
+      .then(data => setReservationData(data.result));
+  }, []);
+
+  const handleCurrentTab = e => {
+    const currentTabName = e.target.innerHTML;
+    const currentTabId = e.target.id;
+    fetch(`/data/Reservation_Data-${currentTabId}.json`)
+      .then(res => res.json())
+      .then(data => setReservationData(data.result));
+    setCurrentTab(currentTabName);
+  };
+
+  const overTen = num => {
+    return num < 10 ? '0' + num : num;
+  };
+
+  return (
+    <S.ReservationData>
+      <S.ReservationTitle>내 여행</S.ReservationTitle>
+      <S.ReservationMain>
+        <S.ReservationLeft>
+          <S.LeftTitle
+            id="1"
+            className={currentTab === '예정된 여행' ? 'active' : ''}
+            onClick={handleCurrentTab}
+          >
+            예정된 여행
+          </S.LeftTitle>
+          <S.LeftTitle
+            id="2"
+            className={currentTab === '지난 여행' ? 'active' : ''}
+            onClick={handleCurrentTab}
+          >
+            지난 여행
+          </S.LeftTitle>
+          <S.LeftTitle
+            id="3"
+            className={currentTab === '취소된 여행' ? 'active' : ''}
+            onClick={handleCurrentTab}
+          >
+            취소된 여행
+          </S.LeftTitle>
```

Comment:
```suggestion
{DATA.map(({ id, title }) => {
            return (
              <S.LeftTitle
                key={id}
                className={currentTab === title ? 'active' : ''}
                onClick={() => handleCurrentTab(id, title)}
              >
                {title}
              </S.LeftTitle>
            );
          })}
```
- 충분히 하실 수 있을 것 같았는데 혹시 추가 기능 구현사항이 있어서 안하신걸까요?

Reply:
뒤에 currentTab에 따라 데이터 요청 기능이 있어서 깜빡했습니다. 바로 수정하겠습니다!!